agent/README.md

```diff
@@ -455,6 +455,26 @@ jar {
 ./gradlew jar -Dbranch=master
 ```
 
+## Multi-project upload
+
+It is possible to upload the same coverage file to multiple Teamscale projects. In this case, 
+the `teamscale-project` property has to be provided in each of the profiled Jar/War/Ear/... files
+via the contained `git.properties` file. For example, the `git.properties` file can be generated
+using the following Gradle plugin:
+
+```groovy
+plugins {
+  id "com.gorylenko.gradle-git-properties" version "2.2.4"
+}
+gitProperties {
+    customProperty 'teamscale-project', 'my-awesome-project' 
+}
+```
+
+Please note that the commit must also be provided via the `git.properties` in each of the profiled Jar/War/Ear...
+files when specifying the `teamscale-project` this way. In case `teamscale-project` is provided both via the agent properties
+and `git.properties`, the value provided via the agent properties takes precedence.
```

Comment:
```suggestion
In case `teamscale-project` is provided both via the agent properties it takes precedence over any `teamscale.project` values specified in the `git.properties` files.
```